radareorg/cutter
# Add menu options for changing IO mode

**Your checklist for this pull request**


**Detailed description**
This PR adds a menu option to set IO modes (write, cache, read only)
![image](https://user-images.githubusercontent.com/20182642/81481797-38afea00-923b-11ea-9942-7d72322c50e9.png)





**Test plan (required)**

1. Make sure that all the options are mutually exclusive. And that the right menu item is disabled when needed.
2. Make sure that when choosing to open in write-mode from the initial analysis dialog, that the menus are set correctly
3. Make sure that writing and caching works correctly
4. Make sure the documentation looks good


**Closing issues**

closes #2127
